### ISSUE TITLE
`--grid-size` is not recognized

### ISSUE THREAD

**Issue by martimpassos** [openvinotoolkit/datumaro#1118]
I'm following the [CLI tiling example](https://openvinotoolkit.github.io/datumaro/latest/docs/reference/_autosummary/datumaro.plugins.tiling.html) quite literally, but it returns `datum: error: unrecognized arguments: --grid-size`, even though running `datum transform -t tile` says that `--grid-size` is required:
```
usage: tile [-h] --grid-size N_ROWS N_COLS [--overlap PERC_HEIGHT PERC_WIDTH] [--threshold-drop-ann THRESHOLD_DROP_ANN]
tile: error: the following arguments are required: --grid-size
```

What am I missing here?

**Comment by martimpassos** [comment]
following the [Python package example](https://github.com/openvinotoolkit/datumaro/blob/c84a13d4882ce9a21aef2608e0fa79642bbc0036/notebooks/06_tiling.ipynb) raises `NotImplementedError` when trying to save the tiled images:
```
Traceback (most recent call last):
  File "/home/ims/.cache/pypoetry/virtualenvs/individualizacao-54J0ewh6-py3.8/lib/python3.8/site-packages/datumaro/plugins/data_formats/coco/exporter.py", line 900, in _apply_impl
    self._save_image(
  File "/home/ims/.cache/pypoetry/virtualenvs/individualizacao-54J0ewh6-py3.8/lib/python3.8/site-packages/datumaro/components/exporter.py", line 277, in _save_image
    item.media.save(path, crypter=crypter)
  File "/home/ims/.cache/pypoetry/virtualenvs/individualizacao-54J0ewh6-py3.8/lib/python3.8/site-packages/datumaro/components/media.py", line 142, in save
    raise NotImplementedError
NotImplementedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/d/dev/individualizacao/tile.py", line 36, in <module>
    tiled_dataset.export(dataset_path + "/tiled", "coco", save_media=True)
  File "/home/ims/.cache/pypoetry/virtualenvs/individualizacao-54J0ewh6-py3.8/lib/python3.8/site-packages/datumaro/util/scope.py", line 158, in wrapped_func
    ret_val = func(*args, **kwargs)
  File "/home/ims/.cache/pypoetry/virtualenvs/individualizacao-54J0ewh6-py3.8/lib/python3.8/site-packages/datumaro/components/dataset.py", line 633, in export
    raise e.__cause__
datumaro.components.errors.ItemExportError: Failed to export item ('P012V001-0237_tile_0', 'default')
```

**Comment by vinnamkim** [comment]
Hi @martimpassos, Thanks for your interest on our project.

Sorry for inconvenience. It seems because we have missed updating tiling part to the latest as well. We acknowledged it and will try to resolve it as soon as possible.

**Comment by vinnamkim** [comment]
There were two problems on this:

1. Saving the tiled image (`RoIImageFromFile`) has not been working properly.
2. Documentation for CLI usage of Tile transform has not been not correct.
 - You should insert a double dash between `-t tile` and `--grid-size`, such as, `datum transform -t tile -- --grid-size 3 2 --overlap 0.1 0.1 --threshold-drop-ann 0.1`. This is because `--grid-size ...` are dedicated arguments for `TileTransform` not for `datum transform`. Please see `datum transform --help` for more details.

Therefore, to fix this issue, we will merge https://github.com/openvinotoolkit/datumaro/pull/1123 to the develop and it will be released to the next version. If you want to use it before the future release, you can install Datumaro from the source with following this guide: https://github.com/openvinotoolkit/datumaro/blob/develop/contributing.md.

Thanks!

**Comment by martimpassos** [comment]
Thanks a lot! Is there an estimative for the next release date?

**Comment by vinnamkim** [comment]
@wonjuleee, Could we make a bug-fix release for this?

**Comment by wonjuleee** [comment]
> @wonjuleee, Could we make a bug-fix release for this?

Thanks @vinnamkim for your quick execution. We plan to have a regular release 1.5.0 in the mid of next month (September). @martimpassos, please wait 1.5.0 or I suggest you to install from the develop branch for a while.

**Comment by vinnamkim** [comment]
Closed this issue since Datumaro 1.5.0 version was released.
